request.js

```diff
@@ -1413,6 +1413,28 @@ Request.prototype.jar = function (jar) {
   return self
 }
 
+Request.prototype.cloneJar = function (cookieJar) {
+  var self = this
+  var newJar, cookieString, cookieStore, domains, domainPaths, cookieUrl
+
+  newJar = self.jar()
```

Comment:
I'm gonna add the tests and then fix this so that I make sure this is done correctly. I'll let you know